src/share/export.js

```diff
@@ -0,0 +1,48 @@
+import fs from '../lib/fs';
+const sh = new fs.Shell();
+import WebTorrent from 'webtorrent';
+
+var client = new WebTorrent();
```

Comment:
Neither of these two variables need to be global, you can move them into your functions below.  Also, prefer `let` and `const` to `var`